tracer/src/Datadog.Trace/Telemetry/Collectors/ConfigurationTelemetryCollector.cs

```diff
@@ -121,13 +122,14 @@ public ICollection<TelemetryValue> GetConfigurationData()
 
             var settings = _settings.Settings;
 
-            var data = new List<TelemetryValue>(settings.IsRunningInAzureAppService ? 31 : 26)
+            var data = new List<TelemetryValue>(27 + (settings.IsRunningInAzureAppService ? 5 : 0))
             {
                 new(ConfigTelemetryData.Platform, value: FrameworkDescription.Instance.ProcessArchitecture),
                 new(ConfigTelemetryData.Enabled, value: settings.TraceEnabled),
                 new(ConfigTelemetryData.AgentUrl, value: settings.Exporter.AgentUri.ToString()),
                 new(ConfigTelemetryData.AgentTraceTransport, value: settings.Exporter.TracesTransport.ToString()),
                 new(ConfigTelemetryData.Debug, value: GlobalSettings.Instance.DebugEnabled),
+                new(ConfigTelemetryData.IsManagedOnly, value: !Instrumentation.ProfilerAttached),
```

Comment:
It would be nice if we only called this once and cached the result somewhere, but I don't expect this to information to change much so I suppose it's okay

Reply:
Yeah, I considered that too, though maybe seems like an optimisation for inside `Instrumentation.Profiler`?